css/at-rules/media.json

```diff
@@ -264,6 +264,61 @@
               "deprecated": false
             }
           }
+        },
+        "any-hover": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/any-hover",
+            "description": "<code>any-hover</code> media feature",
+            "support": {
+              "webview_android": {
+                "version_added": false
+              },
+              "chrome": {
+                "version_added": "41"
+              },
+              "chrome_android": {
+                "version_added": "50"
+              },
+              "edge": {
+                "version_added": "16"
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": false,
+                "notes": "<a href='https://bugzil.la/1035774' rel='noopener'>bug 1035774 – Implement Interaction Media Features including pointer:coarse that replaces non-standard -moz-touch-enabled</a>"
+              },
+              "firefox_android": {
+                "version_added": false,
+                "notes": "<a href='https://bugzil.la/1035774' rel='noopener'>bug 1035774 – Implement Interaction Media Features including pointer:coarse that replaces non-standard -moz-touch-enabled</a>"
+              },
+              "ie": {
+                "version_added": null
+              },
+              "opera": {
+                "version_added": "28"
+              },
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": "9",
+                "notes": "Implemented in <a href='https://bugs.webkit.org/show_bug.cgi?id=134822' rel='noopener'>WebKit bug #134822</a>."
```

Comment:
Would you mind removing this note? Generally, notes that are about historical bugs (i.e., features that have shipped and don't have any known caveats) don't need to be migrated.